rustup-init.sh

```diff
@@ -141,17 +149,17 @@ main() {
     done
 
     if $_ansi_escapes_are_valid; then
-        printf "\33[1minfo:\33[0m downloading installer\n" 1>&2
+        say "\33[1minfo:\33[0m downloading installer\n"
     else
-        printf '%s\n' 'info: downloading installer' 1>&2
+        say '%s\n' 'info: downloading installer'
```

Comment:
It seems that there are many place where this trailing `'%s\n'` doesn't make sense anymore (and is even wrong).

Tried on `dash`:

```bash
warn() {
    printf 'rustup: %s\n' "$1" >&2
}

say() {
    if [ "$RUSTUP_QUIET" = "no" ]; then 
        warn "$1"
    fi
}

warn '%s\n' "Cannot execute (likely because of mounting /tmp as noexec)."
```

This gives `rustup: %s\n`.

Reply:
I'll clean these up as well 👌